### PR TITLE
Update package.py for new magma 2.5.3 release that fixes a bug for CUDA < 9.2

### PR DESCRIPTION
**Pull request by stomov**

### CODE REVIEW

File: var/spack/repos/builtin/packages/magma/package.py [hunk 1]
```diff
@@ -14,9 +14,9 @@ class Magma(CMakePackage, CudaPackage):
     """
 
     homepage = "http://icl.cs.utk.edu/magma/"
-    url = "http://icl.cs.utk.edu/projectsfiles/magma/downloads/magma-2.2.0.tar.gz"
-    maintainers = ['luszczek']
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/magma/package.py]: You accidentally deleted the maintainer

**Comment by v-dobrev** [comment]
Note: this is a PR into the CEED 3.0 PR: #15500.

File: var/spack/repos/builtin/packages/magma/package.py (hunk 1 above)
Comment (stomov) [reply var/spack/repos/builtin/packages/magma/package.py]: Thanks. Actually it wasn't accidental - I wasn't sure of the purpose of the clause as many other packages don't have it, but anyway, I am thinking to maintain it from now on when needed.

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/magma/package.py]: In that case, you can add your name to the list. `maintainers` is a list of GitHub accounts of people who have volunteered to maintain a Spack package. It gives us someone to ping when a user reports a build error, or when someone submits a PR to modify the package.

**Comment by adamjstewart** [comment]
Superseded by #15749

**Pull request closed without merging**